notebooks/multi_modal/solutions/image_captioning.ipynb

```diff
@@ -0,0 +1,1133 @@
+{
```

Comment:
<p>* The note seems weird</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/226/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>I changed the working a bit. Is it clearer now?</p>